sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -1008,7 +1009,7 @@ class Column(val expr: Expression) extends Logging {
   def cast(to: String): Column = cast(CatalystSqlParser.parseDataType(to))
 
   /**
-   * Returns an ordering used in sorting.
+   * Returns a sort expression based on the descending order of the column.
```

Comment:
This and the similar instances below are matched with `functions.scala`. They look calling the same ones.

> Returns a sort expression based on the descending order of the column.

> Returns a sort expression based on the descending order of the column,
> and null values appear before non-null values.

> Returns a sort expression based on the descending order of the column,
> and null values appear after non-null values.